### PR TITLE
feat: [PL-24502]: Rendering module icon in Audit trail based on module field

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
##### Screenshots:

Screenshot 2022-04-12 at 4 59 29 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by avinashmadhwani02** [comment]
dockerize

**Comment by avinashmadhwani02** [comment]
dockerize

**Comment by avinashmadhwani02** [comment]
dockerize

**Comment by avinashmadhwani02** [comment]
dockerzie

**Comment by avinashmadhwani02** [comment]
dockerize

File: src/modules/18-audit-trail/utils/RequestUtil.tsx [hunk 1]
```diff
@@ -61,6 +62,46 @@ export const getModuleNameFromAuditModule = (auditModule: AuditEventDTO['module'
   return undefined
 }
 
+interface ModuleInfo {
+  moduleLabel: StringKeys
+  icon: IconProps
+}
+
+export const moduleInfoMap: Record<AuditEventDTO['module'], ModuleInfo> = {
```
Comment (abhinavrastogi-harness) [inline src/modules/18-audit-trail/utils/RequestUtil.tsx:70]: please add this to the docs... we need to update this map whenever we add a new module

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by mayankVermaHarness — APPROVED** [review]

**Pull request merged**